zips/zip-0312.rst

```diff
@@ -163,44 +159,68 @@ following modifications.
 Randomizer Generation
 '''''''''''''''''''''
```

Comment:
Move this above `Re-Randomized FROST` and make it a same-level section. So this ZIP's Specification section then has:
- Key Generation
- Randomizer Generation
- Re-Randomized FROST
  - Round One - Commitment
  - Round Two - Signature Share Generation
  - Signature Share Verification and Aggregation
- Ciphersuites
  - FROST(Jubjub, BLAKE2b-512)
  - FROST(Pallas, BLAKE2b-512)

which then means that the "Re-Randomized FROST" sub-section is just focused on modifications to FROST, and everything else is stuff canonically specified in ZIP 312 (although maybe parts of "Key Generation" move elsewhere to e.g. ZIP 32, but we'll figure that out in #883).

Reply:
Good point, done in c74bed50